README.md

```diff
@@ -1,34 +1,63 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# SILACAK
+
+A progressive web app (PWA) to track COVID19 cases.
+
+## Purpose and Principles
+
+Help field officer (TNI) to easily tracing the cases. According to WHO Tracing ratio should be 1:15.
+
+### Principles
+
+- Working without reliable connection (even offline mode)
+- New user can input the data easily.
+- Performance
+- Secure
+
+## Developer
+
+Setting Up Silacak in Local.
+
+This is an installation guide for developers before start the project:
+Prerequisite:
+
+1.
 
 ## Getting Started
 
-First, run the development server:
+To get started running the project locally, please follow the steps below.
+
+First, clone the repository:
 
 ```bash
-npm run dev
-# or
-yarn dev
+git clone https://github.com/kawalcovid19/silacak.git
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+Then, install dependencies and fetch data to your local machine:
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+```bash
+cd wargabantuwarga.com
+yarn install
+yarn mirror-box
+```
```

Comment:
Is this part relevant to this project? 🤔

Reply:
I am not sure for this one

Reply:
I think this part is relevant to our other project, the wargabantuwarga.com

Reply:
Exactly, it's irrelevant here. We're neither using the `mirror-box` data nor exporting this site statically.